.mocharc.json

```diff
@@ -2,7 +2,7 @@
   "require": "ts-node/register",
   "extension": ["ts"],
   "recursive": true,
-  "timeout": 5000,
+  "timeout": 60000,
```

Comment:
This file is used for local runs. Increased default timeout here to make IntelliJ happy.

Note that we should really start refactoring our tests soon to move away from using the test tools for sfCommand. With my changes, it should be very easy to construct an action with spys for the Display and test things that way. Also as we pass in our dependencies more and more through the constructor instead of grabbing things out of the global state (singeltons)... then we'll be able to isolate exactly what each test is actually testing... making them more unit like and less end-to-end. I hope we can move towards a future where our end-to-end tests are separate completely and run as a separate step from our unit tests. Only after then can we start decreasing these timeouts again.